jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -167,7 +168,12 @@
     /**
      * The Zipkin B3 trace context propagation format.
      */
-    B3
+    B3,
+
+    /**
+     * The W3C TraceContext propagation format.
+     */
+    TRACE_CONTEXT
```

Comment:
As mentioned in the doc PR, just wondering whether this should be `W3C` rather than trace_context which is very generic. But not sure if there is a convention being used across jaeger (or wider).

Reply:
I don't have any preference. I don't think  there will be any other variant of trace context that will collide with this name.

Reply:
In which case, I would prefer `W3C` as it clearly identifies the standards body - in the same way as `B3` above identifies the defacto-standard.

@yurishkuro Any preference?

Reply:
W3C sgtm, or W3C_TRACE_CONTEXT (but if we also support it in the Config class as a string, then I would just go with `"w3x"` string)

Reply:
> In which case, I would prefer W3C as it clearly identifies the standards body - in the same way as B3 above identifies the defacto-standard.

I don't have much preference but. `B3` identifies the protocol, the same as `trace_context`. Whereas `W3C` is the standard body like or organization which for `B3` is `Zipkin`. So if we follow the same naming conventions the `B3` should be rather named `zipkin`.

Reply:
Agree the analogy was wrong. However the problem I have with `trace_context` is that it is a generic term so could cause confusion.